f_subdyn_test/log.txt

```diff
@@ -0,0 +1,2 @@
+1. vertical cantilever
```

Comment:
In the r-test, use README.md for these kind of files if you can.

Reply:
Yes of course, will formalise with README files once I’m completely satisfied with the results